Configs/.config/hypr/scripts/wallbashcava.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+ScrDir=$(dirname "$(realpath "$0")")
+. $ScrDir/globalcontrol.sh
+CAVA_CONF="/home/khing/.config/cava/config"
+WALL_DCOL="/home/khing/.config/cava/Wall-Dcol"
```

Comment:
Need to change your home dir to $HOME

Reply:
Thanks! I totally forgot to revert while testing